packages/react-native-otlp/android/build.gradle

```diff
@@ -93,11 +90,11 @@ def kotlin_version = getExtOrDefault("kotlinVersion")
 
 dependencies {
   // For < 0.71, this will be from the local maven repo
-  // For > 0.71, this will be replaced by
-  // `com.facebook.react:react-android:$version` (https://central.sonatype.com/artifact/com.facebook.react/react-android)
-  // by react gradle plugin (using currently 0.75.1)
-  implementation "com.facebook.react:react-android:+"
+  // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
+  //noinspection GradleDynamicVersion
+  implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 }
 
-apply from: 'dependencies.gradle'
+// adding Embrace + OpenTelemetry dependencies
+apply from: 'dependencies.gradle'
```

Comment:
btw it might still be better to split these for readable but I think I originally only put this in a separate "dependencies.gradle" file because of having the separate native-src/ directory so could consider re-combining

Reply:
yep, I kept this in a separate file for reusability purposes